manager/manager.go

```diff
@@ -218,7 +221,7 @@ func New(config *Config) (*Manager, error) {
 	m := &Manager{
 		config:      config,
 		listeners:   listeners,
-		caserver:    ca.NewServer(RaftNode.MemoryStore(), config.SecurityConfig),
+		caserver:    ca.NewServer(RaftNode.MemoryStore(), config.SecurityConfig, caReconciliationRetryInterval),
```

Comment:
Sorry, don't follow. Not sure what flaky tests you're referring to or which trick.

Do you think it would be better to have a method on the CA server to change the retry interval, and only use that method from tests? I'd be fine with that.